examples/event-sources/calendar.yaml

```diff
@@ -8,6 +8,13 @@ spec:
       # creates an event every 10 seconds
       interval: 10s
 
+
+#    example-with-catchup:
+#      schedule: "* * * * *"
+#      catchup: true
```

Comment:
I feel like `catchup` is better under `persistence` - `catchup` can not be achieved without `persistence`.

Reply:
Don't forget to update the example.